src/test/java/com/training/assignment/tests/AssignmentTest.java

```diff
@@ -165,4 +165,87 @@ public void testAbstractInterfaceConcept() {
                 .as("Sum of three number is incorrect").isEqualTo(35);
     }
+
+    @Test
+    public void testStaticExample() {
+        String name = "Shri";
+        String lastName = "Shetti";
+        int rollNo = 56;
+        String schoolName = "SVP";
+
+        Student student1 = new Student();
+        Student student2 = new Student();
+
+        student1.setName(name);
+        student1.setLastName(lastName);
+        student1.setRollNo(rollNo);
+        Student.setSchoolName("Anudatt");
+
+        student2.setSchoolName(schoolName);
+
+        softAssert.assertThat(student1.getName()).as("Name is not matching").isEqualTo(name);
+
+        softAssert.assertThat(student1.getLastName()).as("Lastname is not matching").isEqualTo(lastName);
+
+        softAssert.assertThat(student1.getRollNo()).as("Roll number doesn't match").isEqualTo(rollNo);
+
+        softAssert.assertThat(Student.getSchoolName()).as("School name doesn't match").isEqualTo(schoolName);
+
+        softAssert.assertThat(student2.getSchoolName()).as("School name doesn't match").isEqualTo(schoolName);
+    }
+
+    @Test
+    public void testMiniProject() {
+        EmployeeOperation empOperation = new EmployeeOperation();
+
+        String createdName = "Pranav";
+        String updatedName = "Ashwini";
+        String updatedAddress = "Lokhandwala";
+
+        /**
+         * CREATED
+         */
+        empOperation.createEmployee("Shri", "Kandivali", 1000, 1);
+        empOperation.createEmployee(createdName, "Thane", 2000, 2);
+        empOperation.createEmployee("Naresh", "Nerul", 3000, 3);
+        empOperation.createEmployee("Sunil", "panvel", 4000, 4);
+
+        softAssert.assertThat(empOperation.getEmployee().get(0).getId())
+                .as("Id of first employee in list doesn't match with the created employee id")
+                .isEqualTo(1);
+
+        softAssert.assertThat(empOperation.getEmployee().get(1).getId())
+                .as("Id of second employee in list doesn't match with the created employee id")
+                .isEqualTo(2);
+
+        softAssert.assertThat(empOperation.getEmployee().get(2).getId())
+                .as("Id of third employee in list doesn't match with the created employee id")
+                .isEqualTo(3);
+
+        softAssert.assertThat(empOperation.getEmployee().get(3).getId())
+                .as("Id of first employee in list doesn't match with the created id")
+                .isEqualTo(4);
+
+        /**
+         * DELETED
+         */
+        empOperation.deleteEmployee(1);
+
+        softAssert.assertThat(empOperation.getEmployee().get(1).getId() != 2)
+                .as("Employee id 2 is not deleted from list")
+                .isTrue();
+
+        /**
+         * UPDATED
+         */
+        com.training.assignments.miniproject.Employee updateEmployee = empOperation.getEmployee().get(1);
+        updateEmployee.setName(updatedName);
+        updateEmployee.setAddress(updatedAddress);
```

Comment:
Instead of following this design.
Lets add a method in EmployeeOperation class : `updateEmployee(Employee employee)` -> this would update the employee details with the id associated with the employee object - with the other details passed with Employee object.